app/views/components/pageHeader.scala.html

```diff
@@ -36,6 +36,8 @@ <h2 class="govuk-caption-l hmrc-caption-l govuk-!-margin-top-1" id="page-caption
   }
 </h2>
 
-<h1 class="govuk-heading-l govuk-!-margin-bottom-4" id="page-heading">
-  @pageHeader
-</h1>
+@if(showPageHeader) {
```

Comment:
Further to the above, if we change pageHeader to Option[String] we could get rid of the showPageHeader param? What do you think?

Reply:
This check would then become pageHeader.isDefined